internal/store/postgres/resource_repository.go

```diff
@@ -169,7 +169,7 @@ func (r *ResourceRepository) Update(ctx context.Context, res *domain.Resource) e
 	}
 
 	return r.db.WithContext(ctx).Transaction(func(tx *gorm.DB) error {
-		if err := tx.Model(m).Where("id = ?", m.ID).Updates(*m).Error; err != nil {
+		if err := tx.Model(m).Where("id = ?", m.ID).Or("global_urn = ?", m.ID).Updates(*m).Error; err != nil {
```

Comment:
Can you check if this actually works through gorm?

because if I run this directly on postgres, I get an error since the `id` column is of type `uuid`.
query:
```sql
select * from resources 
where id = 'urn:bigquery:project-01:table:project-01:dataset-01' 
or gloabl_urn = 'urn:bigquery:project-01:table:project-01:dataset-01'
```

```error
ERROR: invalid input syntax for type uuid: "urn:bigquery:project-01:table:project-01:dataset-01" Position: 36

Reply:
just realized this, will change the logic